pages/manual/planet.md

```diff
@@ -274,35 +266,25 @@ Surveys are nearly identical to tests in function, but are used for the purpose
 
 ![Survey creation](images/courses-survey.png)
 
-Upon clicking the "Add Question" button, you will find there are four types of questions: 
-
-* Text - Short answer
-
-![Short answer](images/courses-test-short.png)
+Upon clicking the "Add Question" button, you will find there are four types of questions based on two categories __Text__ and __Multiple Choice__:  
 
-Simply fill out the question detail or query for this type of question. The survey taker will automatically get a textbox for filling out his or her answer.
 
-* Text - Long answer
+* Text - Simply fill out the question detail or query for this type of question. The survey taker will automatically get a textbox for filling out his or her answer.  
+	* Short answer: Usually single line short answers.  
+	* Long answer: More possibly to be very long descriptive answers.  
 
+![Short answer](images/courses-test-short.png)
 ![Long answer](images/courses-test-long.png)
 
-Simply fill out the question detail or query for this type of question. The survey taker will automatically get a textbox for filling out his or her answer.
-
-* Multiple Choice - single answer
 
+* Multiple Choice - Click the "Add Choice" button to create a choice of answers for the question. Click the red circle button to the right of the choice to delete it from the question. __Unlike tests, there are no checkboxes denoting the "correct" choice, as this is a survey.__  
+	* Single answer  
+	* Multiple answer  
+	
 ![Multiple choice single answer](images/courses-survey-mc-single.png)
-
-Click the "Add Choice" button to create a choice of answers for the question. Click the red circle button to the right of the choice to delete it from the question. Unlike tests, there are no checkboxes denoting the "correct" choice, as this is a survey.
-
-* Multiple Choice - multiple answer
-
 ![Multiple choice multiple answer](images/courses-survey-mc-multiple.png)
 
-Click the "Add Choice" button to create a choice of answers for the question. Click the red circle button to the right of the choice to delete it from the question.  Unlike tests, there are no checkboxes denoting the "correct" choice, as this is a survey.
-
-While multiple choice questions are automatically graded, text questions must be manually graded.
-
-For multiple choice questions, click the "Add Choice" button to create a choice of answers for the question. Click the red circle button to delete the choice from the question.
+While multiple choice questions are automatically recorded, text questions must be manually recorded.
```

Comment:
This line can be removed. As surveys do not have evaluation. So its almost same for both text and choice questions.

Reply:
Hi @lmmrssa ,
I have removed the line as suggested.
(I have also merged the branch with the latest upstream/master. Hopefully, there will be no conflicts/differences).
Thanks!